### PR TITLE
Allow to restore older volume versions

### LINKED ISSUE #3146: Versioned volume tracings

### PR DESCRIPTION
**Pull request by daniel-wer**
This PR enables the volume version restore view and adds previewing of older volume versions.

### URL of deployed dev instance (used for testing):
- https://volumerestore.webknossos.xyz

### Steps to test:
- Open a volume/hybrid tracing. Do some volume tracing, then open the version view by clicking on "Restore older version" in the dropdown next to the save button.
- Preview older versions of the volume tracing. The segmentation data should live reload.
- Restore an older version. Then volume trace some more and save, everything should work.
- Re-Open the version view, you should see the "Reverted to version x" entry in the version log.

### Issues:
- fixes #3146 

------
- [x] Updated [changelog](../blob/master/CHANGELOG.md#unreleased)
- [x] Updated [documentation](../blob/master/docs) if applicable
- [x] Needs datastore update after deployment
- [x] Ready for review

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -79,7 +79,7 @@
     "gl": "scalableminds/headless-gl#4.0.5"
   },
   "scripts": {
-    "start": "./sbt run -jvm-debug 5005",
+    "start": "./sbt run -jvm-debug 5005 -J-XX:MaxMetaspaceSize=2048m",
```
[the remark this answers is not in the document]
Comment (fm3) [reply package.json]: yarn start is only used locally (and we merged the same change already elsewhere)

File: app/assets/javascripts/oxalis/model/sagas/volumetracing_saga.js [hunk 2]
```diff
@@ -168,6 +171,9 @@ function* labelWithIterator(iterator, contourTracingMode): Saga<void> {
 }
 
 function* copySegmentationLayer(action: CopySegmentationLayerAction): Saga<void> {
+  const allowUpdate = yield* select(state => state.tracing.restrictions.allowUpdate);
```
[the remark this answers is not in the document]
Comment (daniel-wer) [reply app/assets/javascripts/oxalis/model/sagas/volumetracing_saga.js]: #3356 for reference

**Comment by philippotto** [comment]
I just tested it and it works very well :+1: However, I noticed two things:

1) I created a new volume tracing using another segmentation. after doing two brushes and returning to the first version, the first brush was still visible. maybe there is an off-by-one error?
2) When previewing another version and closing the view, that version is still active. Is this intended?

**Comment by daniel-wer** [comment]
> I created a new volume tracing using another segmentation. after doing two brushes and returning to the first version, the first brush was still visible. maybe there is an off-by-one error?

There is a version 0, which is the initial version, but that is not sent by the server, and therefore not yet displayed. I already fixed this on my version view refinement branch :)

create-tracing


> When previewing another version and closing the view, that version is still active. Is this intended?

That is not intentional, I'll have a look.

**Pull request merged**